### DISCUSSION TITLE
How to get a shared folder with mp3s on Synology in Jellyfin Library

### DISCUSSION THREAD

**Discussion by witzker** [jellyfin/jellyfin#8170] [Q&A]
I just installed the Jellyfin APP !! on TrueNas scale

Now I want to add a library that contains music files on a shared folder on Synology:

What has to be prepared on Synology ?
Rights, user, etc ..

What has to be set in Jellyfin to get the folder from Synology with the mp3s in a library?

**Comment by minecraftchest1** [comment]
You would need to make the folder available via a share, ensure there is a user account that can be used to connect to the share, and make sure the user at least has read permissions on the share. Following that, you will need to mount the share on your jellyfin server. Configure jellyfin to use the share (or folders in that share) for one or more of your libraries.
